UnitTest/ApiCompatTests.cs

```diff
@@ -0,0 +1,161 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Numerics;
+using System.Threading;
+using System.Threading.Tasks;
+using Xunit;
+using WinRT;
+
+using WFC = Windows.Foundation.Collections;
+using Windows.Foundation;
+
+using Windows.Devices.Geolocation;
+using Windows.Globalization;
+using Windows.Devices.Enumeration;
+
+using Windows.Data.Json;
+
+using Windows.Storage;
+using Windows.Storage.Streams;
+using Windows.Storage.FileProperties;
+
+using System.Diagnostics;
+using System.Runtime.InteropServices.WindowsRuntime;
+
+namespace UnitTest
+{
+    using A = IEnumerable<IStringable>;
+    using B = KeyValuePair<string, IAsyncOperationWithProgress</*A*/IEnumerable<IStringable>, float>>;
+
+    public class TestAPIs
+    {
+
+        [Fact]
+        public void TestCalendarAPIs()
+        {
+            List<string> languages = new List<string>();
+            languages.Add("en-us");
+            Calendar cal = new Calendar(languages);
+            Assert.NotEqual("", cal.DayOfWeekAsString());
+        }
+
+        async Task InvokeGeolocation()
+        {
+            // Actual position doesn't matter (unless you're on a weather bouy in 
+            // the Gulf of Guinea. Then this might give a false failure). 
+            // This test is just validating the results of calling a simply async OS API.
+            Geolocator locator = new Geolocator();
+            Geoposition pos = await locator.GetGeopositionAsync();

```

Comment:
Hopefully this doesn't throw in the lab due to location being turned off or something, but a run will tell us whether it will or not.

Reply:
Did a run and it seems it does fail.

 Failed UnitTest.TestAPIs.TestGeolocation [22 ms]
  Error Message:
   System.AggregateException : One or more errors occurred. (The service cannot be started, either because it is disabled or because it has no enabled devices associated with it. (0x80070422))
---- System.Runtime.InteropServices.COMException : The service cannot be started, either because it is disabled or because it has no enabled devices associated with it. (0x80070422)
  Stack Trace:
     at System.Threading.Tasks.Task.Wait(Int32 millisecondsTimeout, CancellationToken cancellationToken)
   at System.Threading.Tasks.Task.Wait(Int32 millisecondsTimeout)
   at UnitTest.TestAPIs.TestGeolocation() in D:\a\1\s\UnitTest\ApiCompatTests.cs:line 57

Reply:
Thanks for checking. I wondered about that. Updating to give a :free pass" on this error.